Joplin Server

compgeniuses opened the pull request:
Rebased Adding Joplin App to Nethforge Repo https://github.com/compgeniuses/ns8-joplinserver

stephdl commented:
We have shared together privately and your module have some defaults that you need to fix, this app seems needs some love on the backup of postgres database and maybe other issue, could you ask the test of the community to test this module before to ask to be included inside nethforge

compgeniuses commented:
Thank you for your input,

I have updated the App, and implemented Database dump, as well as database restore, as  required.

Vesion 1.0.1 onward should have this

stephdl commented:
I am closing this pull request for now, like we shared together if you want to apply the guidelines for the Community Certification, please reopen the PR
